chain.go

```diff
@@ -33,7 +33,7 @@ func (c *ChainHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // handler in the order they are passed.
 func chain(middlewares []func(http.Handler) http.Handler, endpoint http.Handler) http.Handler {
 	// Return ahead of time if there aren't any middlewares for the chain
-	if middlewares == nil || len(middlewares) == 0 {
+	if len(middlewares) == 0 {
```

Comment:
makes sense, the nil check is not necessary
https://play.golang.org/p/rj8xKahgRx